class_deque.h

```diff
@@ -0,0 +1,101 @@
+//
+// Created by ifomenko on 27.09.2019.
+//
+
+#ifndef SPISOK_2_CLASS_DEQUE_H
+#define SPISOK_2_CLASS_DEQUE_H
+
+#include "class_myqueue.h"
+
+class myDeque: public myQueue {
+public:
+    myDeque(){};
+    ~myDeque()
+    {
+        while(begin)
+        {
+            List* tmp = begin;
+            begin = begin->next;
+            delete tmp;
+        }
+    };
+
+    void push(int n)
+    {
+        if(end != NULL)
+        {
+            List* temp = new List;
+            temp->num = n;
+            temp->next = NULL;
+            temp->prev = end;
+            end->next = temp;
+            end = end->next;
+        }
+        else
+        {
+            end = new List;
+            end->num = n;
+            end->next = NULL;
+            end->prev = NULL;
+            begin = end;
+        }
+        size++;
+    }
+
+    void push_front(int n)
+    {
+        if(begin != NULL)
+        {
+            List* tmp = new List;
+            tmp->num = n;
+            tmp->prev = NULL;
+            tmp->next = begin;
+            begin->prev = tmp;
+            begin = tmp;
+        }
+        else
+        {
+            begin = new List;
+            begin->num = n;
+            begin->prev = NULL;
+            begin->next = NULL;
+            end = begin;
+        }
+        size++;
+    }
+
+    void pop_back()
+    {
+        try{
```

Comment:
Please use the same {} placement style. (Next line or at line with a function name)